src/main/java/com/example/demo/dao/GradeLocalRepository.java

```diff
@@ -0,0 +1,73 @@
+package com.example.demo.dao;
+
+import com.example.demo.dao.entity.Grade;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@Repository
+public class GradeLocalRepository implements GradeRepository {
```

Comment:
jak tworzymy repository w oparciu o haszmape to najpopularniejsza nazwą jest `InMemory`

np. `InMemoryGradeRepository`